source/Mlos.Python/mlos/unit_tests/TestBayesianOptimizerGrpcClient.py

```diff
@@ -192,4 +194,10 @@ def optimize_quadratic(self, optimizer, num_iterations):
                 registered_objectives_df = objectives_df
             else:
                 registered_objectives_df = registered_objectives_df.append(objectives_df, ignore_index=True)
+
+            best_params, optimum = optimizer.optimum()
+            # ensure current optimum doesn't go up
+            assert optimum.y <= old_optimum

```

Comment:
self.assertTrue(optimum.y <= old_optimum)

Reply:
Is that different? If we move to pytest, the current version will give a nicer error afaik.

Reply:
The unittest way to get the same message would be ``self.assertLessEqual(optimum.y, old_optimum)``